### PR TITLE
Provide more information when a check fails.

### PR DESCRIPTION
**Pull request by rubenvp8510**
@jsanda Do you think we could need more info? like `nodetools` info from Cassandra?

### CODE REVIEW

**Comment by mwringe** [comment]
I am not a big fan of putting this into the Cassandra PV, but it looks like that is the only choice we have at the moment. At least having this data will be a big gain for us.

Do we have any concern that writing the logs may fill up the PV too much? I don't know if we even need the logs at this point TBH, if we get an alert that this check failed, we should be able to go into the cluster and investigate.

**Comment by rubenvp8510** [comment]
@mwringe I was thinking the same, as you said, if something goes wrong, we can get those logs from the cluster, so I think at the end I gonna modify the PR to only store the report with which pods fail and why.

WDYT?

**Comment by rubenvp8510** [comment]
Or is there any concern about losing logs if some C* cluster dies?

**Comment by mwringe** [comment]
We probably do want to handle logs eventually, I think storing the logs is of secondary concern at the moment and what we need now is just the reason for the failures.

File: scripts/monitoring/cron-send-metrics-checks.py [hunk 1]
```diff
@@ -158,6 +160,9 @@ def get_hawkular_creds(self):
             sys.exit(1)
 
     def check_node_metrics(self):
+
+        result_report = {'success':1, 'failed_pods':[] }
+
```
Comment (wshearn) [inline scripts/monitoring/cron-send-metrics-checks.py]: should be after the doc string

File: scripts/monitoring/cron-send-metrics-checks.py [hunk 2]
```diff
@@ -239,13 +289,16 @@ def run(self):
...
+            # persist "false" failure.
+            self.persist_details(metrics_report, pod_report)
             # sleep for 5 seconds, then run the second time node check
             logger.info("The first time metrics check failed, 5 seconds later will start a second time check")
             time.sleep(commandDelay)
             logger.info("starting the second time metrics check")
             metrics_report = self.check_node_metrics()
-
+            # persist second attempt.
+			self.persist_details(metrics_report, pod_report)
```
Comment (wshearn) [inline scripts/monitoring/cron-send-metrics-checks.py]: tab to space

**Comment by rubenvp8510** [comment]
This is ready for another review and merge if it's OK. I could reintroduce the code of storing logs in another PR.

**Comment by wshearn** [comment]
[test]

**Comment by mwringe** [comment]
I think this does what we want. Are we able to easily correlate the alert we get from zabbix to the file we need to check in the Cassandra PV?

**Comment by rubenvp8510** [comment]
I named the file with the date and time `failure_report_{date}_{time}.yaml`. What information do we need to correlate it with Zabbix alert?

**Comment by mwringe** [comment]
What else needs to be done to get this merged?

**Comment by wshearn** [comment]
I found a bug, I am going to talk to rubens today when he gets on.

**Comment by wshearn** [comment]
Closing this PR and opening a new one so the bot is happy,

**Pull request closed without merging**